docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+# Using Gambit with the Prover
+
+This is the mutation verifier which
+ checks that variants of the original
+ solidity program do not pass the specification.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+It also allows users to include manually generated mutants.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Installations and Setup
+
+You will require java to run the mutation testing jar.
+Installation instructions can be found [here](https://www.java.com/en/download/help/download_options.html).
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
```

Comment:
Something important that's missing here is the fact that gambit pulls the configuration for `certoraRun` from a shell script.  This deserves at least a paragraph.

Reply:
ok!